cloudbuild.yaml

```diff
@@ -26,6 +26,9 @@ steps:
   args: ['bash', '-ex', 'run_tests.sh']
   env:
     # Each concurrent test that uses the datastore emulator must have a unique port number
+    # lib-tests is the first test that starts the emulator. The first emulator call creates the
```

Comment:
this is the change in another PR to fix race condition right?

Reply:
Yes, I split this change out into another PR to fix the race condition. I cannot merge that other PR before this one though, as that will overwrite the temporary CI image I am using which has both poetry and Pipenv installed.